docker/build_bionic/Dockerfile

```
@@ -0,0 +1,27 @@
+# A new image must to be pushed, to make changes to this file valid:
+# docker build -t registry.gitlab.com/fritzing/fritzing-app_playground/build:bionic docker/build_bionic
+# docker push registry.gitlab.com/fritzing/fritzing-app_playground/build:bionic
+
+FROM ubuntu:bionic-20190307
+
+MAINTAINER Kjell kjell@gmx.de
+
+RUN apt-get update -y && apt-get install -y build-essential cmake wget git libjpeg-dev flex bison libz-dev libpng-dev libncurses5-dev genparse python-dev libssl-dev libudev-dev qt5-default libqt5serialport5-dev libqt5svg5-dev
```

Please split these on separate lines for better readability.
```suggestion
RUN apt-get update -y \
    && apt-get install -y \
        bison \
        build-essential \
        cmake \
        flex \
        genparse \
        git \
        libjpeg-dev \
        libncurses5-dev \
        libpng-dev \
        libqt5serialport5-dev \
        libqt5svg5-dev \
        libssl-dev \
        libudev-dev \
        libz-dev \
        python-dev \
        qt5-default \
        wget
```